.github/workflows/test.yml

```diff
@@ -83,8 +99,8 @@ jobs:
     - name: Set up Elixir
       uses: erlef/setup-beam@v1
       with:
-        elixir-version: '1.13'
-        otp-version: '24.3'
+        elixir-version: '1.15'
+        otp-version: '25.3'
         version-type: strict
```

Comment:
having an entirely separate job just to check code formatting may be overkill?

unless the idea is to continue-on-error for this particular check, but it's not running with a matrix right now anyway 🤔

Reply:
Pick one matrix (add some attribute to it) to do the formatting and lining and whatnot

Reply:
Okay. I can give that a try. It's going to be a little tricky to know if I got that right on the first try though, given the "workflow awaiting approval" for testing CI.

> 1 workflow awaiting approval

Reply:
Okay. I ran mix format with a newer version of Elixir (as the old lint job was using 1.13)